src/chain_spec.rs

```diff
@@ -131,6 +132,46 @@ pub fn centrifuge_dev(para_id: ParaId) -> CentrifugeChainSpec {
 	properties.insert("tokenSymbol".into(), "DCFG".into());
 	properties.insert("tokenDecimals".into(), 18.into());
 
+	CentrifugeChainSpec::from_genesis(
+		"Centrifuge Dev",
+		"centrifuge_dev",
+		ChainType::Live,
```

Comment:
Why live here?

Reply:
My thought was that when developing locally you should use $runtime-local and when deploying as dev environment we can use the $runtime-dev and with a defined set of authorities. The reason being that ChainType::Local might not behave as expected in deployed environments.